features/vhost/test/capabilities.d/capabilities.list

```diff
@@ -0,0 +1,2 @@
+/usr/bin/ping cap_net_raw=ep
+/usr/lib/(aarch64|x86_64)-linux-gnu/gstreamer1.0/gstreamer-1.0/gst-ptp-helper cap_net_bind_service,cap_net_admin=ep
```

Comment:
For what is `gstreamer` used in the `vhost` feature?

Reply:
`libgstreamer1.0-0` is installed as a dependency by `qemu-system-x86` and `qemu-system-arm`